.github/actions/conda_build_publish_package/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Publish Conda package to Anaconda.org'
+description: 'Build and Publish conda package to Anaconda'
+author: 'Andrew Prokhorenkov, modified by Maxime Borry, modified by Luis Fabregas'
+branding:
+  icon: 'package'  
+  color: 'purple'
+inputs: 
+  subDir:
+    description: 'Sub-directory with conda recipe'
+    default: '.'
+  AnacondaToken:
+    description: 'Anaconda access Token'
+  platforms:
+    description: 'Platforms to build and publish [osx linux win]'
+    default: 'win-64 osx-64 linux-64'
+  python:
+    description: 'Python version to build and publish'
+    default: '3.7'
```

Comment:
It seems you are using 3.8 everywhere else, why 3.7 here?

Reply:
Oh, that is quite an irrelevant line since we do not use the defaults of that action. It would be relevant if I would make a new release of the action for the Marketplace.